### PR TITLE
Gnome tweaks -> 40.0

### PR DESCRIPTION
**Pull request by satmandu**
- Add postinstall to register schemas
- fix pygobject dep

Works properly:
- [x] x86_64

Builds properly:
- [x] x86_64
- [x] armv7l
- [x] i686

### CODE REVIEW

File: packages/gnome_tweaks.rb [hunk 1]
```diff
@@ -2,30 +2,29 @@
...
+  git_hashtag @_ver
 
   binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gnome_tweaks/40.beta-4cbb_armv7l/gnome_tweaks-40.beta-4cbb-chromeos-armv7l.tar.xz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gnome_tweaks/40.beta-4cbb_armv7l/gnome_tweaks-40.beta-4cbb-chromeos-armv7l.tar.xz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gnome_tweaks/40.beta-4cbb_i686/gnome_tweaks-40.beta-4cbb-chromeos-i686.tar.xz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gnome_tweaks/40.beta-4cbb_x86_64/gnome_tweaks-40.beta-4cbb-chromeos-x86_64.tar.xz'
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gnome_tweaks/40.0_armv7l/gnome_tweaks-40.0-chromeos-armv7l.tpxz',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gnome_tweaks/40.0_armv7l/gnome_tweaks-40.0-chromeos-armv7l.tpxz',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gnome_tweaks/40.0_x86_64/gnome_tweaks-40.0-chromeos-x86_64.tpxz'
```
Comment (uberhacker) [inline packages/gnome_tweaks.rb:17]: No i686 binary?

**Comment by satmandu** [comment]
Nope. Gnome on i686 is not something I'm spending time on. Not supported
upstream...

File: packages/gnome_tweaks.rb [hunk 2]
```diff
@@ -2,30 +2,29 @@
...
+  @_ver = '40.0'
   version @_ver
   license 'GPL-3+ and CC0-1.0'
   compatibility 'all'
   source_url 'https://gitlab.gnome.org/GNOME/gnome-tweaks/-/archive/4cbb4a44743b64df5c688e84f28befe28da012ba/gnome-tweaks-4cbb4a44743b64df5c688e84f28befe28da012ba.tar.bz2'
-  # source_url "https://gitlab.gnome.org/GNOME/gnome-tweaks/-/archive/#{@_ver}/gnome-tweaks-#{@_ver}.tar.bz2"
-  source_sha256 '4b0548176772ce999531f6f873ee147420f9dc95980a11a3c2faa052b76119e5'
+  compatibility 'all'
+  source_url 'https://gitlab.gnome.org/GNOME/gnome-tweaks.git'
+  git_hashtag @_ver
```
Comment (supechicken) [inline packages/gnome_tweaks.rb:12]: Do you mean `git_branch @_ver`?

File: packages/gnome_tweaks.rb [hunk 3]
```diff
@@ -2,30 +2,29 @@
 
 class Gnome_tweaks < Package
   description 'Graphical interface for advanced GNOME 3 settings Tweak Tool'
-  @_ver = '40.beta-4cbb'
+  @_ver = '40.0'
   version @_ver
   license 'GPL-3+ and CC0-1.0'
   compatibility 'all'
   source_url 'https://gitlab.gnome.org/GNOME/gnome-tweaks/-/archive/4cbb4a44743b64df5c688e84f28befe28da012ba/gnome-tweaks-4cbb4a44743b64df5c688e84f28befe28da012ba.tar.bz2'
```
Comment (uberhacker) [inline packages/gnome_tweaks.rb:9]: Extra source_url?

Comment (satmandu) [reply packages/gnome_tweaks.rb:9]: yup. sigh.

File: packages/gnome_tweaks.rb (hunk 2 above)
Comment (satmandu) [reply packages/gnome_tweaks.rb:10]: no

File: packages/gnome_tweaks.rb (hunk 1 above)
Comment (satmandu) [reply packages/gnome_tweaks.rb:15]: not building gnome for i686....

**Pull request merged**